src/index.js

```diff
@@ -265,7 +267,7 @@ class TemplateContentScript extends ContentScript {
   }
 
   checkMaintenanceMessage() {
-    const maintenanceMessage = document.querySelector('.big').innerHTML
+    const maintenanceMessage = document.querySelector('.big')?.innerHTML
     if (
       document.location.href === MAINTENANCE_URL &&
       maintenanceMessage === 'Notre site est actuellement en maintenance.'
```

Comment:
I would add a warning with the content if the "big" message if there is a big message which does not meet the criterias of checkMaintenanceMessage.